docs/quickstart/index.md

```diff
@@ -0,0 +1,324 @@
+
+    - [Deploy on Podman](../setting-up/server/podman.md)
+    - [Deploy based on a Docker image](../setting-up/server/docker.md)
+    - [Deploy on Virtual Appliance](../setting-up/server/virtual-appliance.md)
+    - [Deploy on Kubernetes via Helm](../setting-up/server/helm.md)
+    - [Run a PMM instance hosted at AWS Marketplace](../setting-up/server/aws.md)
+
+#### Prerequisites
+
+Before you start installing PMM, verify that your system meets the compatibility requirements.
+
+??? info "Verify system compatibility"
+    - **Disk**: Approximately 1 GB of storage per monitored database node with data retention set to one week. By default, retention is 30 days.
+    - **Memory**: A minimum of 2 GB per monitored database node. The increase in memory usage is not proportional to the number of nodes. For example, the data from 20 nodes should be easily handled with 16 GB.
+    - **Ports**: Your system’s firewall should allow TCP traffic on port 443.
+
+## Install PMM
+
+The Easy-install script only runs on Linux-compatible systems. To use it, run the command with `sudo` privileges or as `root`:
+{ .power-number }
+
+1. Download and install PMM using `cURL` or `wget`:
+
+    === "cURL"
+
+        ```sh
+        curl -fsSL https://www.percona.com/get/pmm | /bin/bash
+        ```
+
+    === "wget"
+
+        ```sh
+        wget -qO - https://www.percona.com/get/pmm | /bin/bash    
+        ```
+
+2. After the installation is complete, log into PMM with the default `admin:admin` credentials.
+
+??? info "What's happening under the hood?"
+     This script does the following:
+
+     * Installs Docker if it is not installed on your system.
+     * Stops and renames any currently running PMM Docker container from `pmm-server` to `pmm-server-{timestamp}`. This old `pmm-server container is not a recoverable backup.
+     * Pulls and runs the latest PMM Docker image.
+
+## Connect database
+
+Once PMM is set up, choose the database that you want it to monitor:
+
+=== "MySQL"
+
+    To connect a Self-hosted MySQL database:
+    { .power-number}
+
+    1. Create database account for PMM using the following command example. This creates a database user with name `pmm`, password `pass`, and the necessary permissions:
+
+        ```sql
+        CREATE USER 'pmm'@'127.0.0.1' IDENTIFIED BY '<your_password>' WITH MAX_USER_CONNECTIONS 10;
+        GRANT SELECT, PROCESS, REPLICATION CLIENT, RELOAD, BACKUP_ADMIN ON *.* TO 'pmm'@'127.0.0.1';
+        ```
+
+    2. Install PMM Client via Package Manager on the database node to optimize server-side resources:
+         
+        1. Install Percona Release Tool:
+
+            ```sh
+            wget https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+            dpkg -i percona-release_latest.generic_all.deb
+            ```
+        2. Install the PMM Client package:
+            
+            ```sh
+            apt update
+            apt install -y pmm2-client
+            ```
+
```

Comment:
I think we should mention that there is another way of installation for red-hat based OSes.